antsibull/build_changelog.py

```diff
@@ -609,7 +609,7 @@ def _get_porting_guide_bytes(changelog: Changelog) -> bytes:
                 f"Ansible {version} is based on Ansible-core {base_version}, which is the same"
                 f" major release as Ansible {prev_version}.  Therefore, there is no section on"
                 " ansible-core in this porting guide.  If you are upgrading from Ansible"
-                f" {prev_prev_version}.x, please first consult the Ansible {prev_version} porting"
+                f" {prev_prev_version}, please first consult the Ansible {prev_version} porting"
```

Comment:
In case someone else looking at this wonders: this message is not supposed to show up with `prev_prev_version == 2`. (That it did show up in the Ansible 4 porting guide was a bug that's now fixed.)